modules/draco/src/lib/draco-parser.js

```diff
@@ -384,15 +384,23 @@ export default class DracoParser {
     }
     const result = {};
     const numEntries = this.metadataQuerier.NumEntries(dracoMetadata);
+    const dracoArray = new this.draco.DracoInt32Array();
     for (let entryIndex = 0; entryIndex < numEntries; entryIndex++) {
       const entryName = this.metadataQuerier.GetEntryName(dracoMetadata, entryIndex);
+      this.metadataQuerier.GetIntEntryArray(dracoMetadata, entryName, dracoArray);
+      const numValues = dracoArray.size();
+      const intArray = new Int32Array(numValues);
+      for (let i = 0; i < numValues; i++) {
+        intArray[i] = dracoArray.GetValue(i);
+      }
       result[entryName] = {
         int: this.metadataQuerier.GetIntEntry(dracoMetadata, entryName),
         string: this.metadataQuerier.GetStringEntry(dracoMetadata, entryName),
         double: this.metadataQuerier.GetDoubleEntry(dracoMetadata, entryName),
-        intArray: this.metadataQuerier.GetIntEntryArray(dracoMetadata, entryName)
+        intArray
```

Comment:
What does this mean? Are we extracting every entry as every type? I assume they can only be of one type? Or they can have several values of different types? 

What happens if they are not of that type? Do we get `null`? 

Some comments please.

Reply:
Also make sure metadata extraction is described in the markdown docs

Reply:
There is no methods to check type of an entry in `MetadataQuerier`. So yes, we extract every entry value as every type. If an entry value of different type - there will be garbage.